force-app/main/default/classes/tasks/BasicTasks.cls

```diff
@@ -15,6 +16,25 @@ public class BasicTasks {
     //write your code below this line
     Double result = 0;
 
+    switch on operator {
+      when '+' {
+        result = a + b;
+      }
+      when '-' {
+        result = a - b;
+      }
+      when '*' {
+        result = a * b;
+      }
+      when '/' {
+        if (b <> 0) {
+          result = a / b;
+        }
+      }
+      when else {
+        System.debug('operator out of task');
```

Comment:
Very nice, I like that you thought about an operator not being on the list of allowed. In a real app you would throw an exception here

Reply:
Re-writen with use IF